orbit-2-viewmodel/orbit-2-viewmodel_build.gradle.kts

```diff
@@ -28,9 +28,18 @@ dependencies {
 
     implementation(ProjectDependencies.androidxLifecycleSavedState)
     implementation(ProjectDependencies.androidxLifecycleKtx)
+    implementation(ProjectDependencies.androidxEspressoIdlingResource)
```

Comment:
I realise google's suggestion is to just have it in your code but some people might not like having this dependency here for their prod code.

Alternative suggestion:

User can add a separate module dependency with this (wrapped) to their preferred flavor/buildtype. Then they call a companion object function to set that as the default idling resource. By default we have the no-op set.

This would mean we can extract idling resource functionality out of the viewmodel module

Reply:
tempted to keep it